### PR TITLE
Close the previous pull request and create a new one

### PR DESCRIPTION
**Pull request by Sethbass**
Hello @bshaffer ,
I have tried the tutorial you sent and got strange result. It was simpler for me to close the previous pull request and to create and new branch.

Thanks

Sebastien

### CODE REVIEW

**Comment by ToddKopriva** [comment]
Thanks, @Sethbass . We'll get this reviewed soon.

File: tutorials/run-symfony-on-appengine-standard/index.md [hunk 1]
```diff
@@ -308,6 +308,51 @@ After following these steps, you can send email messages in `Controller` and `Se
...
+```
+    session:
+        handler_id: Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
+        cookie_secure: auto
+        cookie_samesite: lax
+        # Adjust the max lifetime to your need
+        gc_maxlifetime: 36000
+```
+
+You should then activate the service in `config/services.yaml.
```
Comment (bshaffer) [inline tutorials/run-symfony-on-appengine-standard/index.md:329]: This is missing a backtick after `services.yaml`

Comment (ToddKopriva) [reply tutorials/run-symfony-on-appengine-standard/index.md]: I added it. Thanks, @bshaffer .

**Comment by Sethbass** [comment]
@bshaffer @ToddKopriva  thanks a lot :) Hope everything is ok on your side

**Pull request merged**